src/unity/python/turicreate/visualization/plot.py

```diff
@@ -126,33 +127,44 @@ def save(self, filepath, include_data=True):
         >>> plt.save('vega_spec.json', False)
 
         """
-        spec = _json.loads(self.__proxy__.get('call_function', {'__function_name__': 'get_spec'}))
-
-        if(include_data):
-            data = _json.loads(self.__proxy__.get('call_function', {'__function_name__': 'get_data'}))["data_spec"]
-
-            for x in range(0, len(spec["vega_spec"]["data"])):
-                if(spec["vega_spec"]["data"][x]["name"] == "source_2"):
-                    spec["vega_spec"]["data"][x] = data
-                    break;
+        if type(filepath) != str:
+            raise ValueError("File path provided is not a string")
```

Comment:
I would make it `filepath provided is not a string`. So it matches the parameter name.